### ISSUE TITLE
Looking for Web Developers!

### ISSUE THREAD

**Issue by thecodingwizard** [cpinitiative/usaco-guide#2265]
The Competitive Programming Initiative is looking for web developers to maintain and develop new features. We offer PVSA-certified volunteer hours. If you're interested, [please apply here](https://docs.google.com/document/d/13QpXqdiYQwjBLnywGL1FUG7GFdh8SM_1NigIkJl-A7k/edit?usp=sharing)!

Among other things, you'll be working on:

- The USACO Guide website
- The Competitive Programming Initiative
- Our serverless online judge system
- USACO Guide Groups
- USACO Guide IDE
- Or an idea of your own!

**Comment by SansPapyrus683** [comment]
yeah i think so

**Comment by akshatt877** [comment]
Is the position still available just in case 
  Let me Know if it is available I am willing to work on it

**Comment by bqi343** [comment]
@ryanchou-dev do you know?

**Comment by ryanchou-dev** [comment]
Yes, see the [team application packet](https://docs.google.com/document/d/13QpXqdiYQwjBLnywGL1FUG7GFdh8SM_1NigIkJl-A7k/edit?tab=t.0#heading=h.wqvzgayci72e) for more!

**Comment by piaigmt** [comment]
Hi — I can help with web/front-end maintenance and small scoped features.

Portfolio: https://piaigmt.github.io/
GitHub: https://github.com/piaigmt

Relevant fit: React/static-site work, automation scripts, docs/SEO cleanup, and quick turnaround on small issues. If there is a bite-sized web task that would help the USACO Guide or related CPI tools, I can pick it up and submit a PR.

**Comment by ryanchou-dev** [comment]
> Hi — I can help with web/front-end maintenance and small scoped features.
> 
> Portfolio: https://piaigmt.github.io/ GitHub: https://github.com/piaigmt
> 
> Relevant fit: React/static-site work, automation scripts, docs/SEO cleanup, and quick turnaround on small issues. If there is a bite-sized web task that would help the USACO Guide or related CPI tools, I can pick it up and submit a PR.

That's awesome! Please apply through our linked team application form. We do not accept team members through any other medium.
